open-research.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: default
+title: Open Research
+description: Evidence base and interventions for widening participation in research.
+image: /images/index.jpg
+photos:
+  name: Bérénice Batut
+  license: CC BY-SA 4.0
+  url: https://flic.kr/p/2gHMJah
+---
+
+<!--<h1><figure class="image is-32x32"><img src="{% link images/three-pillars-icons/open-research-icon.png %}"></figure>Open Research</h1>-->
+
+# Open Research
+As researchers, we value evidenced-based behaviours. This means one of our core focuses is creating the community-based, landscape and longitudinal analyses that inform open, equitable research practices. We aim to spread effective techniques and reduce ineffective techniques. Studies are run by experienced hands-on hands-on practitioners of open research and open community.
+
+## Current work streams:
+### Open Seeds Impact Research
+
+Funded by the Wellcome Trust, this study investigates transformative impact of Open Seeds interventions from a qualitative angle. 
+
+Launch announcement: https://openlifesci.org/posts/2021/12/21/wt-open-research-fund/ 
+
+#### Key personnel, impact research: 
+Paz Bernaldo leads the qualitative interview study, and Malvika Sharan is PI. 
```

Comment:
We could add below the people cards

Reply:
I'm pretty certain I replied this comment yesterday. Strange things are happening.

Reply:
I had said:

Adding cards might look aesthetically pleasing. However, it might make it a little difficult for just anyone to contribute to this page. I would suggest that we link directly to their cards on the people page.

What do you think?

Reply:
It looks like this...

![Screenshot (599)](https://github.com/open-life-science/open-life-science.github.io/assets/105166953/fe47450b-9faa-4bab-8618-d1288f68a904)

Reply:
>  However, it might make it a little difficult for just anyone to contribute to this page. 

Do you mean?

Reply:
Depending on how we intend to add the _people_ cards, not everyone will be able to do it.

Reply:
How would you implement it?

Reply:
Put the names in the page's metadata and loop through to display biocards at the bottom of the page. 
That way, a new contributor just needs to add a name to the `persons` array in the metadata.